src/cli.yaml

```diff
@@ -0,0 +1,395 @@
+                        Passing an empty value leaves the metadata empty.
+                multiple: true
+                takes_value: true
+                value_names:
+                    - data
+
+
+    - prepare-metadata:
+        about: Prepare metadata
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+            - metadata:
+                help: Prepare metadata
+                index: 1
+                multiple: true
+
+    - push:
+        about: >
+                 Push all refs associated with issues.
+                 If no issue were supplied, all issues will be pushed.
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+            - remote:
+                help: Push to this remote
+                index: 1
+                multiple: false
+            - issuehash:
+                help: Push this issue
+                index: 2
+                multiple: true
+
+    - reply:
+        about: Reply to a specific message in an issue.
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+            - message:
+                short: m
+                long: message
+                help: Use this as issue message
+                multiple: false
+                takes_value: true
+            - signoff:
+                short: s
+                long: signoff
+                help: Add a 'Signed-off-by' line, with user and email from gitconfig
+                multiple: false
+                takes_value: false
+            - gpgsign:
+                short: S
+                long: gpg-sign
+                help: Add a GPG signature
+                multiple: false
+                takes_value: false
+            - tempfile:
+                short: t
+                long: tempfile
+                help: Use a temporary file at <path> instead of .git/COMMIT_EDITMSG
+                multiple: false
+                takes_value: true
+                value_names:
+                    - path
+            - quote:
+                short: q
+                long: quote
+                help: Quote parent
+                multiple: false
+                takes_value: false
+            - reference:
+                short: e
+                long: reference
+                help: Reference a commit or message in the new message
+                multiple: true
+                takes_value: true
+                value_names:
+                    - commithash
+
+    - show:
+        about: This uses 'git log' to print the issues.
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+            - parent:
```

Comment:
Well, change this to `message`. I suspect that our initial use case for this subcommand somehow slipped into the documentation.